docs/application/vstools/install.md

```diff
@@ -1,170 +1,102 @@
-
-   ![Marketplace](media/howtoinstall-extensionwindows.png)
-
-3. Click **Download** and close the Visual Studio IDE.
-
-   The installation starts.
-
-Alternatively, you can download the Visual Studio Tools for Tizen extension from the [Visual Studio Marketplace Web site](https://marketplace.visualstudio.com/items?itemName=tizen.VisualStudioToolsforTizen).
+## Install Visual Studio Tools for Tizen Extension 
 
-### Installing the Tizen Baseline SDK
+The Visual Studio Tools for Tizen extension is registered in the Visual Studio Marketplace. You can install the extensions from the Visual Studio Marketplace in the Visual Studio IDE.
 
-After installing the Visual Studio Tools for Tizen extension, you must set up the Tizen baseline SDK:
+1.	In the Visual Studio IDE menu, go to **Tools > Extensions and Updates**.
+2.	In the Visual Studio Marketplace, search for **Tizen**. The **Visual Studio Tools for Tizen** option appears. 
+3.	Click **Download** and close the Visual Studio IDE. \
+The installation begins.
 
-1. In the Visual Studio IDE menu, go to **Tools &gt; Tizen &gt; Tizen Package Manager**.
-2. Select **Install new Tizen SDK**.
+![Visual Studio Tools for Tizen](media\v1-install-tools.gif)
 
-   ![Select new installation](media/howtoinstall-installwizard1.png)
+Alternatively, you can download the Visual Studio Tools for Tizen extension from [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=tizen.VisualStudioToolsforTizen).
 
-3. Read the license document and click **I Agree**.
+## Install Tizen Baseline SDK
 
-   ![Agree to license details](media/howtoinstall-installwizard2.png)
+After installing the Visual Studio Tools for Tizen extension, you must set up the Tizen baseline SDK. You can either install a new Tizen baseline SDK or configure an existing Tizen baseline SDK:
 
-4. Enter the root directory path where you want to install and click **Next**.
+- [Install a new Tizen baseline SDK](#install-new-tizen-baseline-sdk)
+- [Configure an existing Tizen baseline SDK](#configure-existing-tizen-baseline-sdk)
 
-   ![Set the installation path](media/howtoinstall-installwizard3.png)
+### Install new Tizen Baseline SDK
 
-   The Tizen SDK installer is downloaded and the baseline SDK is installed automatically.
+1.	In the Visual Studio IDE menu, go to **Tools > Tizen > Tizen Package Manager**.
+2.	Select **Install new Tizen SDK**.
+3.	Read the license document and click **I Agree**.
+4.	Enter the root directory path where you want to install the Tizen baseline SDK and click **Next**. \
+    The baseline SDK is installed automatically and the Tizen Package Manager is launched.
+5.  In the Tizen Package Manager, go to **Main SDK**.
+6.  Install the **Tizen SDK tools**.
 
-   ![Installer download](media/howtoinstall-installwizard4.png)
+![Visual Studio Tools for Tizen](media\v2-install-baseline-sdk.gif)
 
-   ![Baseline SDK installation](media/howtoinstall-installwizard5.png)
+### Configure Existing Tizen Baseline SDK
 
```

Comment:
Information Required: Could you provide some more details to give an overview. Why do we need to install?